utils/mechanical.stanza

```diff
@@ -0,0 +1,268 @@
+    Point(3.35276280472661, 4.84226836073665), 
+    Point(3.31893169638479, 4.86708553613004), 
+    Point(3.28282636825169, 4.88845938385437), 
+    Point(3.24479606476334, 4.90618315610315), 
+    Point(3.2052086505143, 4.92008541200703), 
+    Point(3.16444705192785, 4.93003167596989), 
+    Point(3.12290555323423, 4.93592573844242), 
+    Point(3.08098598258551, 4.93771058655058), 
+    Point(3.03909382519862, 4.93536895557741)])
+  layer("foreground") = Line(1.0, [    
+    Point(1.81836617480138, 4.93536895557741), 
+    Point(1.77647401741449, 4.93771058655058), 
+    Point(1.73455444676577, 4.93592573844242), 
+    Point(1.69301294807215, 4.93003167596989), 
+    Point(1.6522513494857, 4.92008541200703), 
+    Point(1.61266393523666, 4.90618315610315), 
+    Point(1.57463363174831, 4.88845938385437), 
+    Point(1.53852830361521, 4.86708553613004), 
+    Point(1.50469719527339, 4.84226836073665), 
+    Point(1.47346755278222, 4.81424791256009), 
+    Point(1.44514145839502, 4.78329523153066), 
+    Point(1.41999290853756, 4.74970972087191), 
+    Point(1.39826516345885, 4.71381625099316), 
+    Point(1.38016839419095, 4.67596201703966), 
+    Point(1.36587764957855, 4.63651318049705), 
+    Point(1.35553116304318, 4.5958513273357), 
+    Point(1.34922901546048, 4.55436977695498)])
+  layer("foreground") = Line(1.0, [Point(2.86189, 1.88853), Point(2.96349, 1.85615)])
+  layer("foreground") = Line(1.0, [Point(1.99829, 1.51874), Point(2.86189, 1.88853)])
+  layer("foreground") = Line(1.0, [Point(1.99829, 1.9253), Point(1.99829, 1.51874)])
+  layer("foreground") = Line(1.0, [Point(1.99829, 1.41062), Point(2.86189, 1.78041)])
+  layer("foreground") = Line(1.0, [Point(1.89669, 1.443), Point(2.96349, 1.85615)])
+  layer("foreground") = Line(1.0, [Point(2.86189, 1.78041), Point(2.86189, 1.38053)])
+  layer("foreground") = Line(1.0, [Point(2.96349, 1.85615), Point(2.86189, 1.78041)])
+  layer("foreground") = Line(1.0, [Point(1.99829, 2.4333), Point(2.86189, 2.8031)])
+  layer("foreground") = Line(1.0, [Point(1.99829, 2.54142), Point(2.86189, 2.91122)])
+  layer("foreground") = Line(1.0, [Point(1.89669, 2.46568), Point(2.96349, 2.87884)])
+  layer("foreground") = Line(1.0, [Point(1.89669, 2.967), Point(2.96349, 3.38015)])
+  layer("foreground") = Line(1.0, [Point(1.99829, 3.04274), Point(2.86189, 3.41253)])
+  layer("foreground") = Line(1.0, [Point(1.99829, 2.93462), Point(2.86189, 3.30441)])
+  layer("foreground") = Line(1.0, [Point(1.99829, 3.43594), Point(2.86189, 3.80573)])
+  layer("foreground") = Line(1.0, [Point(1.99829, 3.54406), Point(2.86189, 3.91385)])
+  layer("foreground") = Line(1.0, [Point(1.89669, 3.46832), Point(2.96349, 3.88147)])
+  layer("foreground") = Line(1.0, [Point(1.89669, 1.95768), Point(2.96349, 2.37084)])
+  layer("foreground") = Line(1.0, [Point(1.99829, 2.03342), Point(2.86189, 2.40322)])
+  layer("foreground") = Line(1.0, [Point(1.99829, 1.9253), Point(2.86189, 2.2951)])
```

Comment:
How did you generate all this code? Did you use python or something? Same for artworks

Reply:
These are imported geometries